tests/console/verify_separate_home.pm

```diff
@@ -0,0 +1,29 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved. This file is offered as-is,
+# without any warranty.
+
+# Summary: Verification module. Asserts if /home located on the separate
+# partition/volume.
+# Maintainer: Oleksandr Orlov <oorlov@suse.de>
+
+use base "consoletest";
+use strict;
+use warnings FATAL => 'all';
+use testapi;
+
+sub run {
+    select_console 'root-console';
+
+    assert_script_run("lsblk -n | grep '/home'",
```

Comment:
it looks very similar to verify_no_separate_home.pm, did you decide for the moment not create a common class as the test is really short for the moment? It makes sense in that case.

Reply:
Perhaps it would be good to have some class, like Assertions::Partitioning with the human-readable methods like 'assert_separate_home', 'assert_no_separate_home'. WDYT?

Reply:
You already answer my doubt below regarding scheduler and potential parameters. thanks.